src/PublishCodeCovCoverageTask/utils/logger.test.ts

```
@@ -0,0 +1,199 @@
+    expect(tlWarningSpy).toHaveBeenCalledWith('azure warning message');
+  });
+
+  it('should log debug to tl.debug in Azure environment', () => {
+    process.env.AGENT_ID = '1';
+    process.env.NODE_ENV = 'production';
+    logger.debug('azure debug message');
+    expect(tlDebugSpy).toHaveBeenCalledWith('azure debug message');
+  });
+
+  it('should log info to console in non-Azure, non-test environment', () => {
+    delete process.env.AGENT_ID;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)